target/linux/ramips/dts/MAESTRO-E220.dts

```diff
@@ -0,0 +1,112 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "maestro-e220", "ralink,mt7620a-soc";
+	model = "Maestro E220";
+    chosen {
+		bootargs = "console=ttyS0,57600";
+	};
+	gpio-leds {
+		compatible = "gpio-leds";
+		power {
+			label = "maestro-e220:green:power";
+			gpios = <&gpio2 20 1>;
+        };
+		wifi {
+			label = "maestro-e220:blue:wifi";
+			gpios = <&gpio3 0 1>;
+		};
+		network {
+			label = "maestro-e220:orange:network";
+			gpios = <&gpio1 2 1>;
+		};
+		activity {
+			label = "maestro-e220:orange:activity";
+			gpios = <&gpio1 3 1>;
+		};
+		signal {
+			label = "maestro-e220:orange:signal";
+			gpios = <&gpio1 4 1>;
+		};
+		alert {
+			label = "maestro-e220:red:alert";
+			gpios = <&gpio1 5 1>;
+		};
+    };
+};
+
+&gpio0 {
```

Comment:
drop this node. gpio0 should be enabled by default and even if it's not enabled, it isn't used in the dts.